opentracing-jdbc/opentracing-jdbc/src/main/java/org/zalando/opentracing/jdbc/span/ComponentSpanDecorator.java

```diff
@@ -0,0 +1,35 @@
+package org.zalando.opentracing.jdbc.span;
+
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+import org.apiguardian.api.API;
+
+import java.sql.Statement;
+import java.util.List;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+/**
+ * Sets the <code>component</code> span tag, defaults to <code>JDBC</code>.
+ *
+ * @see <a href="https://opentracing.io/specification/conventions/#span-tags-table">Standard Span Tags</a>
+ */
+@API(status = EXPERIMENTAL)
+public final class ComponentSpanDecorator implements SpanDecorator {
```

Comment:
guess this gonna be a first candidate to be moved to some `core` module

Reply:
?